packages/workbox-background-sync/test/browser/request-queue.js

```diff
@@ -106,15 +115,16 @@ describe(`request-queue`, function() {
       expect(callbacks.requestWillEnqueue.calledOnce).to.be.true;
       expect(callbacks.requestWillEnqueue.calledWith(sinon.match.has('request')))
           .to.be.true;
-
+      expect(performance.now() - startTime).to.be.above(800);
```

Comment:
Why was this added and where did 800 come from? This feels super error prone (i.e. Travis is going to be very different to your machine in terms of performance).